services/rule_engine/src/main/java/signals/RequestAvailableDays.java

```diff
@@ -0,0 +1,10 @@
+package signals;
+
+import com.mindsmiths.sdk.core.api.Signal;
+
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
+public class RequestAvailableDays extends Signal {
```

Comment:
Ava should just send available days once it knows them.

Look out - if users says nothing, Ava still needs to send it